### PR TITLE
Feature 1.3.0 prep

### PR DESCRIPTION
**Pull request by JPercival**
This PR fixes some issues found in regression
* Performance issues due to cache misses on Libraries resolved
* A few hackarounds added to address some missing classes in the cql-engine.

### CODE REVIEW

File: evaluator.engine/src/main/java/org/opencds/cqf/cql/evaluator/engine/elm/LibraryMapper.java [hunk 1]
```diff
@@ -55,6 +58,30 @@
                 "unable to map type: " + expression.getClass().getName() + " to ValueSetRef");
     }
 
+    @Mapping(target = "annotation")
+    default List<Object> mapAnnotations(List<Object> annotations) {
+        if ( annotations == null ) {
```
Comment (rob-reynolds) [inline evaluator.engine/src/main/java/org/opencds/cqf/cql/evaluator/engine/elm/LibraryMapper.java:63]: Curious why the extra whitespace...

File: evaluator.engine/src/main/java/org/opencds/cqf/cql/evaluator/engine/elm/LibraryMapper.java [hunk 2]
```diff
@@ -55,6 +58,30 @@
...
+            return null;
+        }
+
+        List<Object> list = new ArrayList<Object>( annotations.size() );
```
Comment (rob-reynolds) [inline evaluator.engine/src/main/java/org/opencds/cqf/cql/evaluator/engine/elm/LibraryMapper.java:67]: And here...

File: evaluator.engine/src/main/java/org/opencds/cqf/cql/evaluator/engine/elm/LibraryMapper.java [hunk 3]
```diff
@@ -55,6 +58,30 @@
...
+        for ( Object object : annotations ) {
```
Comment (rob-reynolds) [inline evaluator.engine/src/main/java/org/opencds/cqf/cql/evaluator/engine/elm/LibraryMapper.java:68]: And here.

**Review by rob-reynolds — APPROVED** [review]

File: evaluator.engine/src/main/java/org/opencds/cqf/cql/evaluator/engine/elm/LibraryMapper.java (hunk 1 above)
Comment (JPercival) [reply evaluator.engine/src/main/java/org/opencds/cqf/cql/evaluator/engine/elm/LibraryMapper.java]: Fixed.

File: evaluator.engine/src/main/java/org/opencds/cqf/cql/evaluator/engine/elm/LibraryMapper.java (hunk 3 above)
Comment (JPercival) [reply evaluator.engine/src/main/java/org/opencds/cqf/cql/evaluator/engine/elm/LibraryMapper.java]: Fixed.

File: evaluator.engine/src/main/java/org/opencds/cqf/cql/evaluator/engine/elm/LibraryMapper.java (hunk 2 above)
Comment (JPercival) [reply evaluator.engine/src/main/java/org/opencds/cqf/cql/evaluator/engine/elm/LibraryMapper.java]: Fixed.

**Pull request merged**